src/main/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformation.java

```diff
@@ -48,6 +57,31 @@ public ReleaseCreationInformation(
         this.pegoutCreationRskTxHash = pegoutCreationRskTxHash;
         this.pegoutBtcTx = pegoutBtcTx;
         this.pegoutConfirmationRskTxHash = pegoutConfirmationRskTxHash;
+
+        this.utxoOutpointValues = this.decodeUtxoOutpointValues(transactionReceipt);
```

Comment:
nit: no need to use this leads to redundancy
```suggestion
        this.utxoOutpointValues = decodeUtxoOutpointValues(transactionReceipt);
```

Reply:
Done